src/resources/views/fields/select2.blade.php

```diff
@@ -14,7 +14,21 @@
         @endif
 
         @if (isset($field['model']))
-            @foreach ($field['model']::all() as $connected_entity_entry)
+            @php
+                if (isset($field['scope'])) {
+                    if (isset($field['scope_params'])) {
+                        $scope_params = $field['scope_params'];
+                    } else {
+                        $scope_params = array();
+                    }
```

Comment:
```php
if (isset($field['scope_params'])) {
    $scope_params = $field['scope_params'];
} else {
    $scope_params = array();
}
```

could be change to a shorter version like this:

`$scope_params = $field['scope_params'] ?? [];`

Or am I wrong?

Reply:
If one wanted to be paranoid:

```
$scope_params = is_array($field['scope_params']) ? $field['scope_params'] : [];
```